.changeset/grumpy-geese-juggle.md

```diff
@@ -0,0 +1,9 @@
+---
+'@finos/legend-application-query-bootstrap': patch
+'@finos/legend-application-studio-bootstrap': patch
+'@finos/legend-art': patch
+'@finos/legend-extension-dsl-mastery': patch
+'@finos/legend-manual-tests': patch
+---
+
+add text support for master record management DSL
```

Comment:
For this, let's have 2 changesets. One changeset should have 
`@finos/legend-extension-dsl-mastery': major`
with the message you have. 
The other could have all the other packages with `patch` and an empty message

Reply:
done